server/router.js

```diff
@@ -10,6 +10,8 @@ const {
   updateGoal,
 } = require('./controllers');
```

Comment:
If you add controllers/serverError within the index.js inside controllers, you can actually just add it the serverError to this destructuring statement!